File: Array_Nama/JavaScript/Array_Nama.js
const namaMahasiswa = ["Ahmad", "John", "Akira", "Beethoven", "Xi Jinping"];
alert("Daftar Mahasiswa:\n" + namaMahasiswa.join(", "));
let namaInput = prompt("Masukkan Nama Mahasiswa: ").toLowerCase();
let ditemukan = false;

for (let i = 0; i < namaMahasiswa.length; i++){
    if(namaMahasiswa[i].toLowerCase() === namaInput){
        alert(`${namaMahasiswa[i]} ada di dalam list`);
        ditemukan = true;
        break;
    }
}
if (!ditemukan){
    alert("Nama tidak ada di dalam list");
}

File: Array_Nama/Java/Array_Nama.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
       String[] namaMahasiswa = {"Ahmad", "John", "Akira", "Beethoven", "Xi Jinping"};
       
       for (String nama : namaMahasiswa){
           System.out.println(nama);
       }
       
       Scanner input = new Scanner(System.in);
       System.out.println("Masukkan Nama Mahasiswa");
       String namaInput = input.nextLine();
       
       boolean ditemukan = false;
       for (String nama : namaMahasiswa){
           if(nama.equalsIgnoreCase(namaInput)){
               System.out.println(nama + " Ada di list");
               ditemukan = true;
               break;
           }
       }
       if (!ditemukan){
           System.out.println("Nama tidak ada di dalam list");
       }
       input.close();
    }
}

